freshrss/umbrel-app.yml

```diff
@@ -0,0 +1,73 @@
+manifestVersion: 1
+id: freshrss
+category: networking
+name: freshrss
+version: "1.21.0"
+tagline: A free, self-hostable aggregator for rss feeds.
```

Comment:
Let's remove the `.` at the end of the sentence for consistency with the rest of the app store

Reply:
removed `.`